src/main/webapp/script.js

```diff
@@ -687,11 +702,12 @@ async function submitRepQuestionnaire() {
         }
     }
     //Remove * on the end of the last platform string if it exists
-    var lastPlatform = listOfPlatforms[listOfPlatforms.length - 1];
-    lastPlatform = (lastPlatform[lastPlatform.length - 1] == "*") ? 
+    if (listOfPlatforms != []) {
```

Comment:
is there an isEmpty or something like that?

Reply:
or rather isNotEmpty?